app/javascript/src/components/Dashboard/Notes/NotesList.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { Clock, MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Typography, Tag, Dropdown, Avatar } from "@bigbinary/neetoui";
+
+export default function NotesList({ notes, index }) {
+  return (
+    <div className="mt-2 flex w-full flex-col">
+      {notes.map(note => (
+        <div
+          className="mb-4 w-full border border-gray-200 bg-white p-6 shadow-md dark:border-gray-700 dark:bg-gray-800"
+          key={index}
```

Comment:
Please understand the purpose of having a `key` prop attached to all list elements. From the looks of how you have structured this component, the `index` argument will always have the same value, and that value has been passed to `key` prop, which would defeat the purpose of having the `key` prop in the first place.

Read:

- https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9
- https://reactjs.org/docs/lists-and-keys.html

Reply:
This is updated afterwards but now have added id on object. Thanks for pointing this out